debian/changelog

```diff
@@ -1,3 +1,25 @@
+cloud-init (22.3.4-0ubuntu1~22.10.1) kinetic; urgency=medium
+
+  * New upstream bugfix release.
+    + Release 22.3.4  (LP: #1986703)
+    + Fix Oracle DS primary interface when using IMDS (LP: #1989686)
+
+ -- Brett Holman <brett.holman@canonical.com>  Fri, 30 Sep 2022 15:41:00 -0600
+
+cloud-init (22.3.3-0ubuntu1~22.10.1) kinetic; urgency=medium
+
+  * New upstream release.
+    + Release 22.3.3 (LP: #1986703)
+    + Fix Oracle DS not setting subnet when using IMDS (#1735)
+      (LP: #1989686)
+    + Release 22.3.2 (LP: #1986703)
```

Comment:
For some reason we are seeing a diff here (and changelog items that were redacted on the previous upload to -proposed)
Have your started his brnach from what was previously released to -proposed. I expect we'd need to start from either `git checkout ubuntu/22.3.4-0ubuntu1_22.10.1` or git checkout upstream/ubuntu/kinetic-22.3-hotfix
```
 cloud-init (22.3.3-0ubuntu1~22.10.1) kinetic; urgency=medium
 
-  * New upstream release.
+  * New upstream bug fix release.
     + Release 22.3.3 (LP: #1986703)
     + Fix Oracle DS not setting subnet when using IMDS (#1735)
       (LP: #1989686)
-    + Release 22.3.2 (LP: #1986703)
     + azure: define new attribute for pre-22.3 pickles (#1725)
     + sources/azure: ensure instance id is always correct (#1727)
       [Chris Patterson]
-    + Release 22.3.1 (LP: #1986703)
 
  -- Brett Holman <brett.holman@canonical.com>  Mon, 19 Sep 2022 13:13:06 -0600
 
```

We don't want to see the deltas for LP: #1986703 in the changelog